spec/src/modules/tracker.js

```diff
@@ -18,7 +18,7 @@ dotenv.config();
 const testApiKey = process.env.TEST_API_KEY;
 const { fetch } = fetchPonyfill({ Promise });
 
-describe('ConstructorIO - Tracker', () => {
+describe.only('ConstructorIO - Tracker', () => {
```

Comment:
Shouldn't be committed - will fix before merge.

Reply:
Looks mostly great.  Might use `emitter` instead of `ee`, tho.  The latter might be too terse for legibility